modules/custom-css/custom-css-4.7.php

```diff
@@ -0,0 +1,498 @@
+
+				$csstidy->parse( $css );
+
+				/**
+				 * Fires after parsing the css with CSSTidy, but only if
+				 * the preprocessor is not configured for use.
+				 *
+				 * @module custom-css
+				 *
+				 * @since 1.7.0
+				 *
+				 * @param obj $csstidy The csstidy object.
+				 * @param array $warnings Array of warnings.
+				 * @param array $args Array of custom CSS arguments.
+				 */
+				do_action( 'safecss_parse_post', $csstidy, $warnings, $args );
+
+				$css = $csstidy->print->plain();
+			}
+		}
+		return $css;
+	}
+
+	/**
+	 * Override $content_width in customizer previews.
+	 */
+	public static function preview_content_width() {
+		if ( isset( $GLOBALS['wp_customize'] ) ) {
+			if ( isset( $_POST['customized'] ) && isset( $_POST['customize_messenger_channel'] ) ) {
+				$customizations = json_decode( stripslashes( $_POST['customized'] ) );
+
+				if ( isset( $customizations->{'jetpack_custom_css[content_width]'} ) ) {
+					$width = (int) $customizations->{'jetpack_custom_css[content_width]'};
+
+					if ( $width ) {
+						$GLOBALS['content_width'] = $width;
+					}
+				}
+			}
+		}
+	}
+
+	static function style_filter( $current ) {
+		if ( is_admin() ) {
+			return $current;
+		} elseif ( self::is_freetrial() && ( ! self::is_preview() || ! current_user_can( 'switch_themes' ) ) ) {
+			return $current;
+		} elseif ( self::skip_stylesheet() ) {
+			/**
+			 * Filter the default blank Custom CSS URL.
+			 *
+			 * @module custom-css
+			 *
+			 * @since 2.2.1
+			 *
+			 * @param string $url Default blank Custom CSS URL.
+			 */
+			return apply_filters( 'safecss_style_filter_url', plugins_url( 'custom-css/css/blank.css', __FILE__ ) );
+		}
+
+		return $current;
+	}
+
+	/**
+	 * Determine whether or not we should have the theme skip its main stylesheet.
+	 *
+	 * @return mixed The truthiness of this value determines whether the stylesheet should be skipped.
+	 */
+	static function skip_stylesheet() {
+		/**
+		 * Prevent the Custom CSS stylesheet from being enqueued.
+		 *
+		 * @module custom-css
+		 *
+		 * @since 2.2.1
+		 *
+		 * @param null Should the stylesheet be skipped. Default to null. Anything else will force the stylesheet to be skipped.
+		 */
+		$skip_stylesheet = apply_filters( 'safecss_skip_stylesheet', null );
+		if ( ! is_null( $skip_stylesheet ) ) {
+			return $skip_stylesheet;
+		}
+
+		$jetpack_custom_css = get_theme_mod( 'jetpack_custom_css', array() );
+		if ( isset( $jetpack_custom_css['replace'] ) ) {
+			return $jetpack_custom_css['replace'];
+		}
+
+		return false;
+	}
+
+	/**
+	 * Override $content_width in customizer previews.
+	 *
+	 * Runs on `safecss_skip_stylesheet` filter.
+	 */
+	public static function preview_skip_stylesheet( $skip_value ) {
+		if ( isset( $GLOBALS['wp_customize'] ) ) {
+			if ( isset( $_POST['customized'] ) && isset( $_POST['customize_messenger_channel'] ) ) {
+				$customizations = json_decode( stripslashes( $_POST['customized'] ) );
```

Comment:
This needs to be replaced with:

```php
$customizations = $GLOBALS['wp_customize']->unsanitized_post_values();
```

Eliminate the `isset( $_POST['customized'] )` check. This will not always be set with changesets.